### PR TITLE
Create solutions to build tools on Windows

### PR DESCRIPTION
**Pull request by semelianova**

### CODE REVIEW

**Comment by dmitry-gurulev** [comment]
@semelianova 
why don't we use cmake to generate all required projects/solutions?

**Comment by dmitryermilov** [comment]
@semelianova  ,please reply to questions reviewers posted

**Comment by semelianova** [comment]
> why don't we use cmake to generate all required projects/solutions?

Now all windows components are built by VS solutions. Cmake is used for Linux only.

File: tools/AllTools.sln [hunk 1]
```diff
@@ -0,0 +1,61 @@
+﻿
```
Comment (dvrogozh) [inline tools/AllTools.sln]: Why do we need tools dedicated solution? We just deleted the one dedicated solution for tutorials. I vote for the single solution entry point - AllBuild.sln

Comment (semelianova) [reply tools/AllTools.sln]: Deleted tools dedicated solution

File: tools/asg-hevc/asg-hevc.sln [hunk 2]
```diff
@@ -0,0 +1,41 @@
+﻿
```
Comment (dvrogozh) [inline tools/asg-hevc/asg-hevc.sln]: @dmitryermilov : do we need this sln? I would drop it as well.

File: tools/bs_parser_hevc/bs_parser_hevc.sln [hunk 3]
```diff
@@ -0,0 +1,31 @@
+﻿
```
Comment (dvrogozh) [inline tools/bs_parser_hevc/bs_parser_hevc.sln]: same

File: tools/bs_parser_hevc/tools/hevc_fei_extractor/hevc_fei_extractor.sln [hunk 4]
```diff
@@ -0,0 +1,41 @@
+﻿
```
Comment (dvrogozh) [inline tools/bs_parser_hevc/tools/hevc_fei_extractor/hevc_fei_extractor.sln]: same

File: tools/asg-hevc/asg-hevc.sln (hunk 2 above)
Comment (dmitryermilov) [reply tools/asg-hevc/asg-hevc.sln]: Well, it's controversial. We have separate solution for samples. Following this logic we need to drop them in samples as well.

Comment (dvrogozh) [reply tools/asg-hevc/asg-hevc.sln]: These are not samples. If we will merge them in we will need to support them. Will we? For example, we don't even CI test these solutions: we use on AllBuild.sln

Comment (dmitryermilov) [reply tools/asg-hevc/asg-hevc.sln]: Why samples require different approach ?

Comment (dvrogozh) [reply tools/asg-hevc/asg-hevc.sln]: They don't, but their solution files existed before. If I would make a decision, I would delete them all, including individual solutions for samples and tutorials.

Comment (dmitryermilov) [reply tools/asg-hevc/asg-hevc.sln]: I see. Makes sense to me.

**Pull request merged**